packages/react/src/progress-indicator/ProgressIndicatorItemButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { ButtonHTMLAttributes } from 'react';
+import { BaseButton } from '../button';
+import {
+	ProgressIndicatorBackground,
+	ProgressIndicatorItemStatus,
+} from './utils';
+import { ProgressIndicatorItem } from './ProgressIndicatorItem';
+
+export type ProgressIndicatorItemButtonProps =
+	ButtonHTMLAttributes<HTMLButtonElement> & {
+		background?: ProgressIndicatorBackground;
+		status: ProgressIndicatorItemStatus;
+		isActive?: boolean;
```

Comment:
Should alphabetise

Reply:
- [ ] alphabetise

Reply:
lol your alphabet must be different than mine 😜

Reply:
- [ ] alphabetise for real